Packages/Models/Sources/Models/Account.swift

```diff
@@ -43,6 +43,14 @@ public struct Account: Decodable, Identifiable, Equatable, Hashable {
   public let bot: Bool
   public let discoverable: Bool?
 
+  public var haveAvatar: Bool {
+    return header.lastPathComponent != "missing.png"
```

Comment:
I believe this should be `avatar.lastPathComponent`?

Reply:
✅